boot/print_icon.h

```diff
@@ -0,0 +1,2627 @@
+// The unique objective of this file is to include
+// instructions to print the illuminatrix icon.
+.altmacro
+.macro PRINT_ICON
```

Comment:
I thing is better idea write a function that get the image.bmp gets every byte and print in the screen, just saying what happen if we need to change the image and we don't have the script, also this is what is making you read all those sectors.

Reply:
the image.bmp should be embedded in the os img of course